parlai/mturk/core/scripts/find_all_hits.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python3
```

Comment:
This file also doesn't need to be added. On all changes mturk-related, we should just use what @JackUrb's been working on.